### DISCUSSION TITLE
Changes to QuickReply

### DISCUSSION THREAD

**Discussion by gazalied** [theforumhelpers/theforumhelpers.github.io#75] [Ideas]
So, Instead of 
> rejected for the following reason: [QUOTE FROM TOLORS].

What about
> rejected for the following reason: [quote=TOLORS]QUOTE FROM TOLORS[/quote].

**Comment by leahcimto** [comment]
@Rendangbike Please post this as an issue on the QuickReply repo here: https://github.com/theforumhelpers/QuickReply
